src/settings/tools/strip-unit.scss

```diff
@@ -1,3 +1,12 @@
+// Ref: https://github.com/sass-collective/sass-collective/blob/master/packages/strip-unit/_functions.scss
+@use "sass:math";
+@use "sass:meta";
+
 @function tbds-strip-unit($value) {
-  @return (calc($value / ($value * 0 + 1)));
+  @if meta.type-of($value) != "number" {
+    @error "Invalid `#{meta.type-of($value)}` type. Choose a number type instead.";
+  } @else if meta.type-of($value) == "number" and not math.is-unitless($value) {
```

Comment:
If the number passed is already unitless should we still return something? Or maybe throw an error/warning (my SASS isn't good enough to know if it has a warning concept)? 

If I were using this function I'd probably like to be told that I don't need to be using it 😆